docs/zkapps/o1js-reference/classes/Unconstrained.md

```diff
@@ -0,0 +1,195 @@
+[o1js](../README.md) / [Exports](../modules.md) / Unconstrained
```

Comment:
@halsaphi 

See https://github.com/o1-labs/docs2/wiki/o1js-Reference

> The zkApp Developers > [o1js Reference](https://docs.minaprotocol.com/zkapps/o1js-reference) docs are generated content and are outside the scope of the Mina Docs repository.

It is safe to remove all generated content updates from this PR, since the docs will auto-generate in the main branch. See https://github.com/o1-labs/docs2/blob/main/CONTRIBUTING.md#o1js-reference